spec/features/catalog_view_spec.rb

```diff
@@ -133,6 +133,10 @@
       expect(page).to have_selector(".document-access.review", text: "Restrictions may apply. See Access Note.")
       expect(page).to have_selector("#component-summary .document-access.restricted", text: "Restricted Content")
     end
+    it "lets someone click on the access restrictions badge", js: true do
+      visit "/catalog/AC136_c2889"
+      click_on "Restrictions may apply. See Access Note."
```

Comment:
Can you add an assertion here so we know what it should be displaying after the click?